invenio_rdm_records/records/systemfields/access/grants.py

```diff
@@ -76,7 +89,7 @@ def subject_type(self):
         else:
             # if it's nothing else, it's gonna be a system role
             # (which doesn't have a persisted subject entity)
-            return "sysrole"
+            return "system_role"
```

Comment:
renamed `sysrole` to `system_role`, to better match the [`SystemRole` need](https://github.com/inveniosoftware/invenio-access/blob/master/invenio_access/permissions.py#L31)